### ISSUE TITLE
Compare paper results with running with train/validation/test and train/test split and early stopping

### ISSUE THREAD

**Issue by Nielsmitie** [Nielsmitie/DM_Lab#34]
Rerun
- [ ] Agnos-S
- [ ] Agnos-G
- [ ] Agnos-W
with 
- [ ] only training data.
- [ ] training data and evaluating on test data
- [ ] training data with validation set and early stopping evaluated on test set

For all runs calculate average and std and compare the results.
